contracts/contracts/BLSWallet.sol

```diff
@@ -202,7 +202,7 @@ contract BLSWallet is Initializable, IBLSWallet
         try this._performOperation(op) returns (
             bytes[] memory _results
         ) {
-            success = (_results.length > 0); // false when no actions given
```

Comment:
@jzaki Can you provide more context on why this was changed from feedback from the last audit? https://github.com/jzaki/bls-wallet/commit/3bbcdb29e7de1e48d23a5634df06449611d4db2c

Reply:
@jzaki Covered this in the PR description:

> If requiring actions to be non-empty is desirable, then I suggest adding require(op.actions.length > 0); to _performOperation (I don't think it's desirable though - we actually already do this in testing).

There's a use case where an empty action list can be used as a no-op to increment your nonce (or if you just want your wallet contract to be deployed). But for me it's just as much about consistency. Inconsistency leads to mistakes. `[].every(x => x)` is `true`, not `false`, because every element of that empty list is true.

Reply:
Thanks @jzaki & @voltrevo for more context, I think it's fine to allow no-ops.